cli/src/pcluster/config/update_policy.py

```diff
@@ -244,12 +244,29 @@ def is_login_fleet_stop_required_for_shared_storage_change(change, patch):
     return patch.cluster.has_running_login_nodes() and not is_login_fleet_update_supported_for_shared_storage(change)
 
 
-def is_login_fleet_update_supported_for_shared_storage(_):
+def is_login_fleet_update_supported_for_shared_storage(change):
     """
-    Check if the live update of the running login fleet is supported for the given change.
+    Check if the live update of the running login fleet is supported for the given change to SharedStorage.
 
-    In particular, it's never supported.
+    We are referring here to a live update, that does not require the replacement of login nodes.
+    In particular, the live update is supported only in the following cases:
+      1. mount/unmount of external EFS
+      1. mount/unmount of external FSx
```

Comment:
So specifically external EBS and other supported storage options are not supported?  Just EFS and FSx?

Reply:
Yes, this is the MVP for the next release.
To be clear: here we are talking about **live updates**, that will be limited to external EFS/FSx*; a drain/terminate update is instead already supported.